### ISSUE TITLE
🐛 Bug Report: System message content is not saved for Anthropic

### ISSUE THREAD

**Issue by dinmukhamedm** [traceloop/openllmetry#2187]
### Which component is this bug for?

Anthropic Instrumentation

### 📜 Description

Contents of the system message are not instrumented for calls to Anthropic via Anthropic Python SDK.

### 👟 Reproduction steps

1. Enable Traceloop instrumentation with `Traceloop.init()`
2. Send a call to Anthropic via their Python SDK with `system` enabled. See example below.
3. Observe resulting spans in Traceloop or another observability platform of your choice.

Example:
```python
from traceloop.sdk import Traceloop
from anthropic import Anthropic

Traceloop.init()
anthropic = Anthropic()

anthropic.messages.create(
    model="claude-3-5-sonnet-20240620",
    max_tokens=1024,
    system=[
        {
            "type": "text",
            "text": "You are a helpful and friendly assistant",
        },
    ],
    messages=[
        {
            "role": "user",
            "content": "Tell me a joke about LLM developers",
        },
    ],
)

```

### 👍 Expected behavior

System message must be saved in attributes `gen_ai.prompt.0.content` and `gen_ai.prompt.0.role`, at least for consistency with OpenAI instrumentation.

### 👎 Actual Behavior with Screenshots

System messages are actually ignored and not added to the span.

### 🤖 Python Version

3.9

### 📃 Provide any additional context for the Bug.

I am assuming the main reason for this difference is that system messages are passed inside the message array in OpenAI, but as a separate request field for Anthropic.

I know that this might not be the priority now that stuff is moving towards events, but I think the migration to events will take some time, and would like system messages to be added for now.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by nirga** [comment]
@dinmukhamedm this is fixed with #2238 right?

**Comment by dinmukhamedm** [comment]
Yes @nirga, sorry, forgot to close it.
